3.2/root/usr/share/container-scripts/mongodb/init-petset-replset.sh

```diff
@@ -134,6 +134,7 @@ function add_member() {
     info "ERROR: couldn't join to replica set!"
     info "CAUSE: failed after waiting for becoming a member. Command output was:"
     echo "${rs_status_out}"
+    echo "==> End of the error output <=="
```

Comment:
I want to have it in the image because it will be very useful if it fails. Otherwise it's mostly impossible to understand where command's output is ending.